### ISSUE TITLE
Add "forward chamfer distance" in pytorch3d.loss.chamfer

### ISSUE THREAD

**Issue by JiahaoPlus** [facebookresearch/pytorch3d#502]
## 🚀 Feature

Forward chamfer distance (chamfer distance only from source to target)

## Motivation

Currently, in pytorch3d.loss.chamfer, the chamfer_distance function can returns "cham_dist = cham_x + cham_y". However, in many cases, we only need "forward chamfer distance", i.e., "cham_x" in the code. 

## Pitch

It would be great to have a parameter (i.e., direction = "forward" or "bidirectional") to choose which to return (cham_x or cham_x + cham_y).

**Comment by gkioxari** [comment]
The definition of chamfer is what you call here "bidirectional". If you want to compute only the one-directional distance, this should likely be called something else, not chamfer. We can add such a function for convenience.

**Comment by JiahaoPlus** [comment]
Yes, agree. Thanks for your effort!

**Comment by low5545** [comment]
Any updates on when this loss function will be released?
